From 5e637305a0dc82c5f797159721c5606896712045 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Pablo=20Correa=20G=C3=B3mez?= <ablocorrea@hotmail.com>
Date: Thu, 19 Sep 2024 10:05:06 +0200
Subject: [PATCH 4/4] plugins: appstream: make refine found flag local to app
 loop

This is also some small fallout from
5dd59c4a4752fab9e5e83c498dfc32a33263edba

Discovered by Milan Crha
---
 plugins/core/gs-plugin-appstream.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/core/gs-plugin-appstream.c b/plugins/core/gs-plugin-appstream.c
index aa39f94d2..7ccbac12a 100644
--- a/plugins/core/gs-plugin-appstream.c
+++ b/plugins/core/gs-plugin-appstream.c
@@ -1167,7 +1167,6 @@ refine_thread_cb (GTask        *task,
 	GsPluginRefineData *data = task_data;
 	GsAppList *list = data->list;
 	GsPluginRefineFlags flags = data->flags;
-	gboolean found = FALSE;
 	g_autoptr(GsAppList) app_list = NULL;
 	g_autoptr(GHashTable) apps_by_id = NULL;
 	g_autoptr(GHashTable) apps_by_origin_and_id = NULL;
@@ -1251,6 +1250,7 @@ refine_thread_cb (GTask        *task,
 	}
 
 	for (guint i = 0; i < gs_app_list_length (list); i++) {
+		gboolean found = FALSE;
 		GsApp *app = gs_app_list_index (list, i);
 
 		/* not us */
-- 
2.46.1

